MusicDB/1 table.sql

```diff
@@ -0,0 +1,56 @@
+Use MusicDB
+go
+drop table if exists instrument
+go 
+create table dbo.instrument(
+InstrumentID int not null identity primary key,
+CustomerFirstName varchar(15) not null
+    constraint ck_Instrument_Customer_first_name_cannot_be_blank check (CustomerFirstName > ''), 
+CustomerLastName varchar(15) not null
+    constraint ck_Instrument_Customer_last_name_cannot_be_blank check (CustomerLastName > ''), 
+Instrument varchar(20) not null
```

Comment:
-- LL This should have a clearer column name - InstrumentName? because your table itself is called instrument